draft-ietf-masque-h3-datagram.md

```diff
@@ -973,12 +738,7 @@ STREAM(44): HEADERS            -------->
   Sec-Use-Datagram-Contexts = ?1
```

Comment:
```suggestion
  sec-use-datagram-contexts = ?1
```

Reply:
And other locations

Reply:
This isn't specific to this PR, I'll take care of it in a followup